test/internal/SetupResumeAcceptorTest.cpp

```diff
@@ -138,56 +135,9 @@ TEST(SetupResumeAcceptor, SingleSetup) {
   EXPECT_TRUE(setupCalled);
 }
 
-TEST(SetupResumeAcceptor, SetupAndFnf) {
```

Comment:
Why did you delete this one?

Reply:
This test is invalid, actually.  It sends a SETUP and a REQUEST_FNF frame to a `OneFrameProcessor`, which isn't supposed to happen.